File: module_b/src/main/java/com/github/xiaogegechen/module_b/presenter/IFragmentBPresenter.java
package com.github.xiaogegechen.module_b.presenter;

import android.os.Parcelable;
import com.github.xiaogegechen.common.base.IBasePresenter;
import com.github.xiaogegechen.module_b.view.IFragmentBView;

public interface IFragmentBPresenter extends IBasePresenter<IFragmentBView> {
    /**
     * 跳转到星座详情Activity
     * @param param 星座名和iconId
     */
    void gotoDetailActivity(Parcelable param);
}

File: module_b/src/main/java/com/github/xiaogegechen/module_b/presenter/IFragmentBPresenter.kt
package com.github.xiaogegechen.module_b.presenter

import com.github.xiaogegechen.common.base.IBasePresenter
import com.github.xiaogegechen.module_b.view.IFragmentBView

interface IFragmentBPresenter:IBasePresenter<IFragmentBView> {

    // 查询今日运势
    fun queryToday(name: String)

    // 查询本周运势
    fun queryWeek(name: String)

    // 查询年度运势
    fun queryYear(name: String)

    // 加载背景图
    fun queryImage()

}